app/assets/javascripts/action_tracker.js

```diff
@@ -2,27 +2,37 @@
 var ActionTracker = function () {
 
   var storage,
+      time_seed,
+      options = {},
       callbacks = {};
 
   function constructor() {
     storage = new Storage();
+    time_seed = new TimeSeed();
   }
 
   function setCallbacks(callbacksObj) {
     callbacks = callbacksObj;
   }
 
-  function start(list) {
+  function start(list, cfg_options) {
+    if(typeof cfg_options !== 'undefined') options = cfg_options;
     storage.queue(list);
   }
 
   function process() {
     while(typeof storage.getFirst() !== 'undefined') {
-      var tracker = new Tracker(storage.dequeue());
-      tracker.send();
+      var tracker = new Tracker(storage.dequeue(), trackerOptions());
+      tracker.addTimestamp(time_seed).send();
```

Comment:
Wouldn't be better if we check the timestamp options here? This way, ```addTimestamp``` will only know about adding timestamp. Also it keeps easier to extract is decision logic out of here, in a further moment. What do you think?